getting-started/display.rst

```diff
@@ -49,6 +49,12 @@ Next, start up a Python interpreter.
 You can use the default Python shell (:command:`python`), the `IPython shell`_, or even run from a `Jupyter Notebook`_.
 Ensure that this Python session is running from the shell where you ran :command:`setup lsst_distrib`.
 
+Some user-reported solutions to ds9-related issues include:
+
+* Launching ds9 *after* setting up the LSST environment in a new terminal window.
+* If using Ubuntu, installing ``xpa-tools`` with ``sudo apt install xpa-tools; xpans``. 
```

Comment:
This prerequisite should be added to the installation documentation (https://pipelines.lsst.io/install/prereqs.html#debian-ubuntu)

Reply:
Added, thanks